ballerina/tests/mock_service.bal

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+
+listener http:Listener httpListener = new (9090);
+
+http:Service mockService = service object {
+    resource function post chat/completions(@http:Payload CreateChatCompletionRequest payload) returns CreateChatCompletionResponse|http:BadRequest {
+
+        // Validate the request payload
+        if (payload.messages[0]["content"].toString() is "" || payload.model.toString() is "") {
```

Comment:
Usually, we don't use parentheses for if statements. 
https://learn-ballerina.github.io/best_practices/avoid_parentheses.html

Reply:
wasn't aware of that. I'll fix it.

Reply:
wasn't aware of that. I'll fix it.